PhilipChat/src/chat.css

```diff
@@ -0,0 +1,39 @@
+.chat-page-content {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.message {
+  font-size: 1rem;
+  display: flex;
+  flex-direction: column;
+  border: solid black 1px;
+  background-color: lightblue;
+  border-radius: 50px;
+  padding: 20px;
+}
+
+.author {
+  background-color: lightgreen;
+}
+
+.user-author {
+  display: flex;
+  justify-content: flex-start;
+  color: black;
+}
+
+.user-message {
+  display: flex;
+  justify-content: flex-end;
+  color: grey;
+}
+
+.message-button {
+  display: flex;
+  flex-direction: row;
+  gap: 200px;
```

Comment:
Thats a big gap, maybe use margins or paddings instead for spacing items apart from each other.